co/component.py

```diff
@@ -328,7 +328,7 @@ def mutate(self, mutations, strict=True):
 
         for feature in changed_features:
             logging.debug('changed: %s', feature)
-            translated_feature = Component._translate_feature(feature, component, tt)
+            translated_feature = translate_function(feature, component, tt)
```

Comment:
I see what the issue is here, but this is probably the wrong approach. We should refactor the code so that changed features are tracked separately and then at the end there is one step where the features are transformed. (transform? great name.)